app/Data/Articles/ArticleData.php

```diff
@@ -47,7 +47,7 @@ public static function fromModel(Article $article): self
             category: $article->category,
             content: $article->content,
             image: $article->getMedia()->first()->getUrl(),
-            publishedAt: (int) $article->published_at->timestamp,
+            publishedAt: (int) $article->published_at->getTimestampMs(),
```

Comment:
why was this change needed?

Reply:
because JS expects timestamp to be in milliseconds

Reply:
usually we do `* 1000` in the frontend to achieve that (e.g. `ResultStep.tsx` and various other places). Might be best to keep the timestamp as-is and handle it in the frontend to keep it consistent

Reply:
Fixed. Thanks